index.html

```diff
@@ -107,8 +107,25 @@ <h3 class="work-title">work 1</h3>
         </section>
         <section class="testimonial">
              <h1 class="testimonial-title underline-center">Take Their Words</h1>
-             <div class="testimonials-container" id="testimonialCard">
-               
+             <div class="testimonials-container">
+                <div class="testimonial-card">
+                    <div class="testimonial-card-img">
+                    </div>
+                    <div class="testimonial-card-content">
```

Comment:
- [ ] Kindly follow closely [the design guideline](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
- [ ] When you see your design you have not added the icons for Facebook and Twitter.

- Your design
<img width="1411" alt="Screenshot 2023-05-31 at 05 28 36" src="https://github.com/habitoz/serategna/assets/81444338/ec8b9040-c3e8-4bb0-9ac5-6fdebab39068">

Reply:
> * [ ]  Kindly follow closely [the design guideline](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
> * [ ]  When you see your design you have not added the icons for Facebook and Twitter.
> * Your design
> 
> <img alt="Screenshot 2023-05-31 at 05 28 36" width="1411" src="https://user-images.githubusercontent.com/81444338/242275433-ec8b9040-c3e8-4bb0-9ac5-6fdebab39068.png">

As requested by the previous reviewer, kindly stick to the design as closely as possible. You seem to be missing some very important designs on the page. It's required to customize the content such as text and images but keep to the design.

Design Guide:
![image](https://github.com/habitoz/serategna/assets/20554583/86974a74-668c-4e1b-a491-7c93e0681a79)